### DISCUSSION TITLE
What is next, mDNS

### DISCUSSION THREAD

**Discussion by lathoub** [sstaub/Ethernet3#37] [Ideas]
Hi,
In your "What is next", you mention that you are working on mDNS - great! 
Any work you can share? Happy to help.
[EthernetBonjour3 ](https://github.com/michael71/EthernetBonjour3) works for a while, but is not stable over time - before i dive in, maybe you have something to start from)
Best

**Comment by sstaub** [comment]
In the moment I have no time to work on it. If there are problems with [https://github.com/michael71/EthernetBonjour3](url) please make an issue there. The plan was to integrate that library. I hope I will have some time to make an update to the W6100 chip, but the support of WIZnet to support for Arduino is horrible because the official Arduino library is pure chaos.
